ice/mvc/model.zep

```diff
@@ -299,7 +299,7 @@ abstract class Model extends Arr implements \Serializable
             let this->messages = [];
         }
 
-        this->di->applyHook("model.before.validate", [this]);
+        this->di->applyHook("model.before.validate." . static::$class, [this]);
```

Comment:
How about `this->from`?

Reply:
`this->from` is not safe enough. people may create two or more models class to operate the same table